public/app/features/plugins/sql/components/visual-query-builder/SQLWhereRow.tsx

```diff
@@ -31,6 +32,18 @@ export function SQLWhereRow({ query, fields, onQueryChange, db }: WhereRowProps)
       config={{ fields: state.value || {} }}
       sql={query.sql!}
       onSqlChange={(val: SQLExpression) => {
+        const templateSrv = getTemplateSrv();
+        const templateVars = templateSrv.getVariables() as VariableWithMultiSupport[];
```

Comment:
It's odd that this is passing the drone build, I thought we disallowed all type assertions in new code? Or am I crazy @mdvictor ?

Reply:
Type assertions _do_ normally make the build fail Betterer tests - which I was concerned about too. Victor and I looked it over together and decided this was the only reasonable solution, so we updated the Betterer file to add this to the list of accepted failures, so it now passed.